packages/button/docs/overview.mdx

```diff
@@ -109,14 +109,7 @@ A button should have a [meaningful text label](/packages/forms/button/rationale#
 For example, using a magnifying glass icon to indicate to a user they can search for something on mobile.
 
 ```jsx live
-<ButtonLink
```

Comment:
Updating this button icon example so it doesn't include an external link icon. The usage of this should be discouraged as `(opens in new tab)` would not get announced to screen readers.